packages/breadcrumb/src/react/index.tsx

```diff
@@ -0,0 +1,58 @@
+import Button from '@pluralsight/ps-design-system-button'
+import { CaretLeftIcon } from '@pluralsight/ps-design-system-icon'
+import { RefForwardingComponent } from '@pluralsight/ps-design-system-util'
+import { css } from 'glamor'
+import React from 'react'
+
+import stylesheet from '../css'
+
+const styles = {
+  breadcrumb: () => css(stylesheet['.psds-breadcrumb'])
+}
+
+interface BreadcrumbProps
+  extends Omit<
+    React.HTMLAttributes<HTMLDivElement>,
+    'disabled' | 'href' | 'onClick'
+  > {
+  disabled?: boolean
+  href?: string
+  loading?: boolean
+  onClick?: (
+    evt: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement, MouseEvent>
+  ) => void
+}
+
+interface BreadcrumbStatics {}
+
+interface BreadcrumbComponent
+  extends RefForwardingComponent<
+    BreadcrumbProps,
+    HTMLAnchorElement | HTMLButtonElement,
+    BreadcrumbStatics
+  > {}
+
+const Breadcrumb = React.forwardRef<
+  HTMLAnchorElement | HTMLButtonElement,
+  BreadcrumbProps
+>((props, ref) => {
+  const { disabled, href, loading, onClick, ...rest } = props
+  return (
+    <div {...rest} {...styles.breadcrumb()}>
```

Comment:
I know rest was probably here to begin with but I can't help but feel it should be on button. It's the interaction trigger not the div.  We should drop the padding on the wrapping div and add margins to button instead of padding on the wrapping div. We could even move the line-height to the button.